docs-source/usersguide/library/02_compiling.rst

```diff
@@ -325,69 +325,50 @@ You should see a series of test results like this:
 
-Congratulations! You successfully have built and installed OpenMM from source.
+Congratulations! You have successfully built and installed OpenMM from source.
+
 
 Building the Documentation (Optional)
 *************************************
 
-The documentation that you're currently reading, as well as the developer guide and API
-documentation can be built through CMake by setting the OpenMM option :code:`OPENMM_GENERATE_API_DOCS=ON`.
-
-User Guide and Developer Guide
-==============================
-
-Generating the user guide and developer guide requires the following dependencies
-
-* Sphinx (http://sphinx-doc.org/)
-
-* sphinxcontrib-bibtex (https://pypi.python.org/pypi/sphinxcontrib-bibtex)
-
-These dependencies may not be available in your system package manager, but should
-be installable through Python's ``pip`` package manager. ::
-
-   pip install sphinx sphinxcontrib-bibtex
-
-The developer and user guides can be built either as HTML or a PDFs. Building the
-PDF version will also require a functional LaTeX installation.
-
-To build the HTML version of the documentation, type: ::
-
-  make sphinxhtml
-
-To build the PDF version of the documentation, type: ::
-
-  make sphinxpdf
-
-
-Python and C++ API Documentation
-================================
+The documentation that you're currently reading, as well as the Developer Guide and API
+documentation, can be built through CMake.  To do that, you need to install a few
+additional tools.  The easiest way is to use :code:`pip` to install them into
+your Python environment.  The following command installs everything needed to
+build documentation in HTML format.
+::
 
-The following dependencies are required to build the Python and C++ API documentation.
+    pip install sphinx sphinxcontrib-bibtex sphinxcontrib-lunrsearch sphinxcontrib-autodoc_doxygen jinja2
 
-* Sphinx (http://sphinx-doc.org/)
+To build documentation in PDF format, you also must have a functional LaTeX
+installation.  It can be obtained from https://www.latex-project.org/get.
 
-* sphinxcontrib-lunrsearch (https://pypi.python.org/pypi/sphinxcontrib-lunrsearch)
+If you want to build documentation, make sure that OPENMM_GENERATE_API_DOCS is
+set to ON when configuring the build in CMake.
 
-* sphinxcontrib-autodoc_doxygen (https://pypi.python.org/pypi/sphinxcontrib-autodoc_doxygen)
+To build the documentation, use the following build targets.
 
+* :code:`sphinxhtml`: Build the User Guide and Developer Guide in HTML format.
 
-These dependencies may not be available in your system package manager, but should
-be installable through Python's ``pip`` package manager. ::
+* :code:`sphinxpdf`: Build the User Guide and Developer Guide in PDF format.
 
-   pip install sphinx sphinxcontrib-lunrsearch sphinxcontrib-autodoc_doxygen
+* :code:`C++ApiDocs`: Build the C++ API documentation.
 
-To build the C++ API documentation, type: ::
+* :code:`PythonApiDocs`: Build the Python API documentation.
```

Comment:
These instructions work for me for all targets except `PythonApiDocs`, which needs the `install` target to have been run previously. We should either fix this dependency or document it here.

```suggestion
* :code:`PythonApiDocs`: Build the Python API documentation. This requires that ``make install`` has already been run.
```